pokemonCards/pokemoncard_api/serializers.py

```diff
@@ -30,4 +30,14 @@ class PokemonCardWriteSerializer(serializers.ModelSerializer):
     class Meta:
         model = PokemonCard
         fields = ('pokedexIndex','name', 'HP', 'pokemonCardSet', 'type')
-        
\ No newline at end of file
+
+class PokemonCollectionReadSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PokemonCollection
+        fields = '__all__'
+        depth = 2
+
+class PokemonCollectionWriteSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PokemonCollection
+        fields = '__all__'
```

Comment:
looks like this wants a newline at the end of the file. So just hit enter after this line, save the file and Push that up